src/test/java/edu/kis/vh/nursery/IntArrayStackTest.java

```diff
@@ -0,0 +1,104 @@
+package edu.kis.vh.nursery;
+
+import edu.kis.vh.nursery.storage.IntArrayStack;
+import edu.kis.vh.nursery.storage.IntStorageInterface;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class IntArrayStackTest {
+
+    @Test
+    public void isEmptyShouldReturnTrue(){
+
+        IntArrayStack intArrayStack = new IntArrayStack();
+        final boolean EXPECTED_VALUE = true;
+
+        Assert.assertEquals(EXPECTED_VALUE,intArrayStack.isEmpty());
+
+    }
+
+    @Test
+    public void isEmptyShouldReturnFalse(){
+
+        IntArrayStack intArrayStack = new IntArrayStack();
+        final boolean EXPECTED_VALUE = false;
+
+        intArrayStack.push(1);
+
+        Assert.assertEquals(EXPECTED_VALUE,intArrayStack.isEmpty());
+
+    }
+
+    @Test
+    public void isFullShouldReturnTrue(){
+
+        IntArrayStack intArrayStack = new IntArrayStack();
+        final boolean EXPECTED_VALUE = true;
+
+        for(int i=0; i<IntArrayStack.getCAPACITY(); i++){
+            intArrayStack.push(i);
```

Comment:
Original test asserted isFull state before pushing element (this would show an error)